frontend/src/component/admin/apiToken/ProjectsList/ProjectsList.tsx

```diff
@@ -1,10 +1,11 @@
+import { Fragment, type FC } from 'react';
 import { styled } from '@mui/material';
 import { Highlighter } from 'component/common/Highlighter/Highlighter';
 import { HtmlTooltip } from 'component/common/HtmlTooltip/HtmlTooltip';
 import { LinkCell } from 'component/common/Table/cells/LinkCell/LinkCell';
 import { TextCell } from 'component/common/Table/cells/TextCell/TextCell';
 import { useSearchHighlightContext } from 'component/common/Table/SearchHighlightContext/SearchHighlightContext';
-import { Fragment, type FC } from 'react';
+import ErrorIcon from '@mui/icons-material/ReportGmailerrorredRounded';
```

Comment:
I've never seen this icons before in our code. Don't we have an established icon for warnings or errors (if we think it's an error)?